starters/solidjs-tanstack-tailwind/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
```

Comment:
editorconfig also allows us to set `end_of_line`. Shall we add that with the value of "lf"?